library/ssl_misc.h

```diff
@@ -1323,4 +1366,29 @@ static inline int mbedtls_ssl_conf_is_hybrid_tls12_tls13( const mbedtls_ssl_conf
 }
 #endif /* MBEDTLS_SSL_PROTO_TLS1_2 && MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL*/
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+
+static inline void mbedtls_ssl_handshake_set_state( mbedtls_ssl_context* ssl,
+    int state )
```

Comment:
Yes I think it would be better.

Reply:
Could I keep this in this PR ? For tls13_only and tls12_only mode , with new field should be OKay . But for tls13 and tls12 both mode, it might be problem.

Reply:
Thanks for the heads-up. That's ok the way it is currently, resolving the conversation.